scripts/python/idc_index_builder.py

```diff
@@ -0,0 +1,11 @@
+import os
+from idc_index_data_manager import IDCIndexDataManager
+
+project_id = os.environ["GCP_PROJECT_ID"]
+manager = IDCIndexDataManager(project_id)
+latest_idc_release_version = manager.retrieve_latest_idc_release_version()
+manager.update_sql_queries_folder(
+        "scripts/sql/", latest_idc_release_version
+    )
```

Comment:
```suggestion
```

`run_queries_folder` should probably be an implementation details, ideally the function `generate_index_data_files` would be the one called, it would accept one parameter named `idc_version`

Reply:
using only one function from now.